flexmeasures/ui/utils/view_utils.py

```diff
@@ -86,14 +87,27 @@ def render_flexmeasures_template(html_filename: str, **variables):
         options["downloadFileName"] = f"asset-{asset.id}-{asset.name}"
     variables["chart_options"] = json.dumps(options)
 
-    variables["menu_logo"] = current_app.config.get("FLEXMEASURES_MENU_LOGO_PATH")
+    account: Account | None = (
```

Comment:
These changes (about logo) should also be on main already

Reply:
They aren't on main yet, I pulled the changes from the `add_logo` branch since I needed to make the `logo` field also editable.